frontend/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script src="http://localhost:8097"></script>
```

Comment:
Why do we need this line? What is its purpose?

Reply:
As you see it's outdated I missed removing it And did remove it afterwards. I just recently tried to squash the commits, but I am not profficient with GitHub :D